src/Chainweb/Miner/Core.hs

```diff
@@ -86,20 +85,18 @@ usePowHash Development f = f $ Proxy @Blake2s_256
 usePowHash Testnet04 f = f $ Proxy @Blake2s_256
 usePowHash Mainnet01 f = f $ Proxy @Blake2s_256
 
--- | This Miner makes low-level assumptions about the chainweb protocol. It may
--- break if the protocol changes.
+-- | CPU POW mining for chainweb.
 --
--- TODO: Check the chainweb version to make sure this function can handle the
--- respective version.
```

Comment:
This TODO is outdated. With the advent of ASICs, the chainweb work header format is final and won't change.